Framework/PythonInterface/mantid/api/CMakeLists.txt

```diff
@@ -125,6 +129,8 @@ list (APPEND ALL_FILES ${EXPORT_FILES} ${SRC_FILES})
 enable_precompiled_headers(inc/MantidPythonInterface/api/PrecompiledHeader.h ALL_FILES)
 
 # Create the target for this directory
+set(FRAMEWORK_DIR ../../..)
+include_directories(${FRAMEWORK_DIR}/CurveFitting/inc)
```

Comment:
Better to use target_include_directories, but I'm changing the cmake framework code in #32066, so I can update it.

Reply:
If you can update it I will leave it as is for now.